src/libsystemd-network/ndisc-option.c

```diff
@@ -1267,6 +1279,147 @@ static int ndisc_option_build_prefix64(const sd_ndisc_option *option, usec_t tim
         return 0;
 }
 
+int ndisc_option_add_encrypted_dns_internal(
```

Comment:
I think now that https://github.com/systemd/systemd/pull/31807 is merged `ndisc_option_add_*` should be accompanied by `ndisc_option_build_*`. For example in https://github.com/systemd/systemd/commit/9f3587ae44ec9b437bd124eb9abbd593af45a796 `ndisc_option_add_home_agent` came with `ndisc_option_build_home_agent` and `ndisc_send` was updated too.

(It should help the fuzz target a bit too)

Reply:
I did see that, but thought that, since I don't implement the server part, if added it would just end up unused. I didn't realize it was used by the fuzz target. I will add this function.